<launch>
<!--
	 <node name="ped_momdp" pkg="ped_is_despot" type="ped-momdp-sarsop" launch-prefix="gdb -ex run - -args" respawn="false" output="screen" ns="/scooter">
	 -->
	 <node name="ped_pomdp" pkg="ped_is_despot" type="ped_pomdp"  respawn="false" output="screen" required="true">
	    <rosparam file="$(find ped_is_despot)/params_initial_belief2.yaml" command="load" />
		<remap from="pomdp_path_repub" to="new_global_plan"/>
		<remap from="navgoal" to="/move_base_simple/goal"/>
        <remap from="robot_0/amcl_pose" to="amcl_pose"/>
        <!-- <remap from="odom" to="odom"/> -->
        <param name="simulation" value="false"/>
	 </node>

	 <node name="ped_momdp_local_transform" pkg="ped_is_despot" type="local_frame"  >
	    <param name="global_frame" value="/map"/>
        <param name="threshold" value="0.0"/>
	</node>

	 <node name="vel_publisher" pkg="ped_is_despot" type="vel_publisher" output="screen" >
	    <!--<remap from="cmd_vel" to="cmd_vel"/>-->
	 </node>
         
        <!-- <node name="record" pkg="rosbag" type="record" args="-a"/> -->

	<!-- <node pkg="ped_is_despot" type="record.sh" name="bag_recorder" cwd="node"/> -->
	 <!--<node pkg="ped_is_despot" type="plot.sh" name="velplot" cwd="node"/>-->
</launch>
